lib/src/dropdown_button2_data.dart

```diff
@@ -132,6 +133,16 @@ class DropdownStyleData {
   /// The inner padding of the dropdown menu
   final EdgeInsetsGeometry? padding;
 
+  /// The outer horizontal margin of the dropdown menu
+  /// Only works when [MenuItemStyleData.width] is set.
+  ///
+  /// Total width of the dropdown menu will be equal to
+  /// [MenuItemStyleData.width] - 2* [outerHorizontalMargin]
+  ///
+  /// It allows to center the menu dropdown when using [DropdownDirection.center]
+  /// with additional padding to it's parent.
+  final double? outerHorizontalMargin;
```

Comment:
@pawlowskim 
I added the review but forgot to submit it, sorry 🤦

Reply:
It's easier to use, instead setting the margin manually. It's optional, so both ways works. But yea, it might be an overkill to add property just for that.

Reply:
This will complicate things imo. Also, It should cover vertical margin and only left/right margin too.

User can easily do:
- width: screenWidth - someMargin
- maxHeight: screenHeight - someMargin